plugins/main/dummy/dummy.go

```diff
@@ -0,0 +1,309 @@
+// Copyright 2022 Arista Networks
```

Comment:
typo: Copyright

Reply:
I think you need something GPL/MIT/APACHE ?

Reply:
Apache license follows